examples/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "tinyman-js-sdk-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start:dev": "nodemon"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "algosdk": "^1.22.0",
+    "@tinymanorg/tinyman-js-sdk": "^2.0.1"
```

Comment:
What do you think about getting the package from `../dist` instead? We can always get the latest version that way. 
Also, maybe we can create a pre-commit script (or we can simply run the tests of the examples project on main projects pre commit hook) that runs the `main` function and if it throws an error we can cancel the commit process. So every time we make a change we can be sure the examples are working. @jamcry  @edizcelik

Reply:
If we want to treat these examples as tests, these are good suggestions 👍 I don't think we should treat these examples as tests, and pressure ourselves with this 😄 We don't have a dedicated team for SDK, and most of the time our first focus is get the changes done on SDK, and get it working on web app ASAP. So sometimes when we make changes, we might not have time to update examples (similar to the case for v2 launch)

> So every time we make a change we can be sure the examples are working.

Yes this is a very valid concern. I think we can accomplish this by being careful if we will make changes on the examples (I don't think we will do big changes, since they use the core functionality), and leaving the dependency with the exact version.

However (I think we discussed this recently), we can plan to start writing tests for the main repo. We can start incrementally (we can add tests when we add/change something.) cc @edizcelik Maybe we can ticket this, and start slowly this or next week:
- Install test libraries and configure
- For starters, add tests for a small file (maybe for example pool utils)

Reply:
Creating the tickets for setting up a testing environment and adding simple tests:
https://linear.app/hipo/issue/TINY-904/js-sdk-test-configuration
https://linear.app/hipo/issue/TINY-905/js-sdk-add-tests-for-basic-utility-functions